spec/frameSpec.js

```diff
@@ -0,0 +1,12 @@
+'use strict';
+describe('Frame', function() {
+  var frame;
+  beforeEach(function() {
+    frame = new Frame();
+  });
+  it('allows two rolls', function() {
+    frame.roll(3)
+    frame.roll(5)
+    expect(frame.rolls).toEqual([3, 5])
```

Comment:
I like how you do it 1 roll at a time rather than frame, this allows for easy implementation of the frame and bonuses. Nice planning!